addons/calendar/models/calendar_event.py

```diff
@@ -293,7 +293,7 @@ def _compute_is_highlighted(self):
     def _compute_attendee(self):
         for meeting in self:
             attendee = meeting._find_my_attendee()
-            meeting.attendee_status = attendee.state if attendee else 'needsAction'
+            meeting.attendee_status = attendee and attendee.state
```

Comment:
what if the attendee is not found? The status must have a default value here as a fallback.
Note: the default value is needsAction in the attendee.state definition

Reply:
It is to differentiate "needsAction" that means we are waiting on the decision of the attendee from when there is no decision to wait

If we don't find the attendee, there is no attendee_status for the meeting at all
and this helps us to determine if the attendee was invited, he might be in needsaction in which case he is attendee, but if there is no attendee_status, then he is not attending